Jenkinsfile

```diff
@@ -4,9 +4,7 @@ library("govuk")
 
 node("postgresql-9.6") {
   govuk.buildProject(
-    postgres96Lint: false,
     rubyLintDiff: false,
```

Comment:
I think we can remove this line too: https://github.com/alphagov/govuk-jenkinslib/pull/57